samples/batch/transcription-enabled-storage/Connector/Serializable/TranscriptionFiles/TranscriptionFiles.cs

```diff
@@ -0,0 +1,19 @@
+﻿// <copyright file="TranscriptionFiles.cs" company="Microsoft Corporation">
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE.md file in the project root for full license information.
+// </copyright>
+
+namespace Connector.Serializable.TranscriptionFiles
+{
+    using System.Collections.Generic;
+
+    public class TranscriptionFiles
+    {
+        public TranscriptionFiles(List<TranscriptionFile> values)
+        {
+            Values = values;
+        }
+
+        public List<TranscriptionFile> Values { get; }
```

Comment:
better use IEnumerable or IList as return value

Reply:
Updated it!